src/main/java/it/gov/pagopa/reward/notification/service/email/EmailNotificationServiceImpl.java

```diff
@@ -0,0 +1,92 @@
+package it.gov.pagopa.reward.notification.service.email;
+
+import it.gov.pagopa.reward.notification.connector.email.EmailNotificationRestClient;
+import it.gov.pagopa.reward.notification.connector.selc.SelcRestClient;
+import it.gov.pagopa.reward.notification.dto.email.EmailMessageDTO;
+import it.gov.pagopa.reward.notification.dto.selc.UserResource;
+import it.gov.pagopa.reward.notification.model.RewardOrganizationImport;
+import it.gov.pagopa.reward.notification.repository.RewardNotificationRuleRepository;
+import it.gov.pagopa.reward.notification.utils.PerformanceLogger;
+import it.gov.pagopa.reward.notification.utils.Utils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@Slf4j
+public class EmailNotificationServiceImpl implements EmailNotificationService{
+
+    public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+    private final String commaDelimiter;
+    private final EmailNotificationRestClient emailRestClient;
+    private final SelcRestClient selcRestClient;
+    private final RewardNotificationRuleRepository notificationRuleRepository;
+
+    public EmailNotificationServiceImpl(@Value("${app.email-notification.comma-delimiter}") String commaDelimiter,
+                                        EmailNotificationRestClient emailRestClient,
+                                        SelcRestClient selcRestClient,
+                                        RewardNotificationRuleRepository notificationRuleRepository) {
+        this.commaDelimiter = commaDelimiter;
+        this.emailRestClient = emailRestClient;
+        this.selcRestClient = selcRestClient;
+        this.notificationRuleRepository = notificationRuleRepository;
+    }
+
+    @Override
+    public Mono<RewardOrganizationImport> send(RewardOrganizationImport organizationImport, String templateName, String subject) {
+
+        return PerformanceLogger.logTimingOnNext(
+                "FEEDBACK_ELABORATION_NOTIFICATION",
+                getInstitutionProductUsers(organizationImport.getOrganizationId())
+                        .map(UserResource::getEmail)
+                        .collectList()
+                        .map(l -> String.join(commaDelimiter, l))
```

Comment:
comma-delimiter is already describing that the separator would be a comma :D
maybe it's better tu user just delimiter or separator as name (also in the application.yml)